CargoWebAsset.js

```diff
@@ -43,64 +38,42 @@ class CargoWebAsset extends Asset {
         }
     }
 
-    async install_nightly() {
-        let [stdout] = await exec( "rustup", ["show"] );
-        if( !stdout.includes( "nightly" ) ) {
+    static async install_nightly() {
+        const rustup_show = await exec( "rustup show" );
+        if( !rustup_show.stdout.includes( "nightly" ) ) {
             await pipeSpawn( "rustup", ["update"] );
             await pipeSpawn( "rustup", ["toolchain", "install", "nightly"] );
         }
     }
 
-    async install_cargo_web() {
-        if( cargo_web_command ) {
-            return;
-        }
-
-        if( process.env.CARGO_WEB ) {
```

Comment:
Out of curiosity, will this ever be anything other than `cargo-web`?

Reply:
In this plugin - most likely not. This is here so that you can override the `cargo-web` it uses through an environment variable for e.g. testing.